config/application.rb

```diff
@@ -49,8 +50,16 @@ class Application < Rails::Application
 
     config.assets.enabled = true
 
-    # TODO: move to :zeitwork mode
```

Comment:
Is this comment worth keeping? To go from classic to zeitwerk eventually

Reply:
We have a ticket to do this work anyway, which i think suffices. we have too many TODOs scattered around the code base generally and I had added this one as part of this config work earlier and don't think its worth keeping now.